src/style.css

```
@@ -94,19 +94,73 @@
+  background-image: linear-gradient(90deg, rgba(108, 75, 204, 0.25) 1px, transparent 1px),
+                    linear-gradient(0deg, rgba(108, 75, 204, 0.25) 1px, transparent 1px);
+  background-size: 50px 50px;
+  pointer-events: none;
+  mask-image: radial-gradient(circle at 50% 40%, rgba(0, 0, 0, 1) 0%, rgba(0, 0, 0, 0) 75%);
+  -webkit-mask-image: radial-gradient(circle at 50% 40%, rgba(0, 0, 0, 1) 0%, rgba(0, 0, 0, 0) 75%);
+  z-index: -1;
+  filter: blur(4px);
+  animation: gridMove 24s linear infinite, gridPulse 8s ease-in-out infinite alternate;
+}
+
+@keyframes gridMove {
```

Stylelint (17.14.0)

[error] Expected keyframe name "gridMove" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)